infra/terraform/modules/data_upload_user/iam.tf

```diff
@@ -0,0 +1,26 @@
+resource "aws_iam_user" "system" {
+  name = "${var.system_name}"
+  path = "/uploaders/${var.org_name}/"
```

Comment:
That's because the name attribute of the IAM user is the system name, so the final ARN will end with `:role/uploaders/hmpps/nomis`, whereas the policy's name reflects the permissions granted, so the ARN there will end with `:policy/uploaders/hmpps/nomis/s3_upload_writeonly`

Reply:
Oh, right, sorry that makes perfectly sense 👍